Lab 1/Lab1/Validation/ValidatorInputField.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Lab1.LogsAndExceptions;
+
+namespace Lab1
+{
+    public class ValidatorInputField
+    {
+        public static void ValidateType(string obj, bool mustBeString)
+        {
+            if (mustBeString)
```

Comment:
String shouldn't be validated in this way.
Just use string.IsNullOrWhiteSpace().

Reply:
Remove string validation from the method.